platform_design/lib/widgets.dart

```diff
@@ -235,27 +235,27 @@ class SongPlaceholderTile extends StatelessWidget {
                   Container(
                     height: 9,
                     margin: EdgeInsets.only(right: 60),
-                    color: Colors.grey[300],
+                    color: Theme.of(context).disabledColor,
```

Comment:
nit: Since these rows are stand-ins for text, something like `Theme.of(context).textTheme.body1.color` might be more appropriate asa color choice.

Reply:
Actually, by using the text theme's color the views looked too bright and didn't go well with the dark theme. Nonetheless, I understand that they are stand-ins for text and using the text theme's color would be better for people looking at this repo. I've made the changes according to your comment. Please take a look.